implementation/src/main/java/io/smallrye/jwt/auth/cdi/RawClaimTypeProducer.java

```diff
@@ -123,10 +122,10 @@ Double getClaimAsDouble(InjectionPoint ip) {
 
     @Produces
     @Claim("")
-    public Optional getOptionalValue(InjectionPoint ip) {
+    public <T> Optional<T> getOptionalValue(InjectionPoint ip) {
```

Comment:
@sberyozkin - I think this may be the cause of the Quarkus error. I'll revert it.

Reply:
Cool, thanks for spotting it